src/emb.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+from fastapi import Depends
+from pydantic import BaseModel
+from langchain.docstore.document import Document
+from langchain.embeddings import HuggingFaceEmbeddings
+from langchain.vectorstores import Chroma
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+class SpliteDoc(BaseModel):
+    '''
+    [
+        Document(page_content="..."),
+        Document(page_content="...")
+    ]
+    '''
+    texts : List[Document]

```

Comment:
```suggestion
    texts : list[Document]

```

Python 3.9 이상 부터는 `List -> list`, `Dict -> dict`, `Tuple -> tuple` 타이핑으로 사용하시면 됩니다.